Wrappers/Python/cil/framework/__init__.py

```diff
@@ -1,5 +1,5 @@
-#  Copyright 2019 United Kingdom Research and Innovation
-#  Copyright 2019 The University of Manchester
+#  Copyright 2024 United Kingdom Research and Innovation
+#  Copyright 2024 The University of Manchester
```

Comment:
Please revert any changes to the year, our convention is for the copyright statement to include the year of first creation of the file.

Reply:
Files should only be marked as 2024 if they contain *only* new code (e.g. base.py), otherwise they should be marked with the copyright year of the file the code was originally located in (probably framework.py)

Reply:
what if a file used to say "copyright 2020 UKRI" which Manchester makes a massive modification to in 2024? You aren't allowed to add "copyright Manchester 2020" just to keep the dates the same, and Manchester will insist on saying "copyright Manchester 2024".